src/web.js

```diff
@@ -23,17 +23,25 @@ document.body.appendChild(style);
 const elements = document.querySelectorAll('.likecoin-embed.likecoin-button');
 elements.forEach((div) => {
   const likerId = div.getAttribute('data-liker-id');
-  if (!likerId) {
+
+  const href = div.getAttribute('data-href') || window.location.href;
+
+  const iscnId = div.getAttribute('iscn-id');
```

Comment:
I think adding `data-` as prefix is common for params

Reply:
Reference:
https://developer.mozilla.org/zh-TW/docs/orphaned/Web/API/HTMLOrForeignElement/dataset

Reply:
So you can access with `div.dataset.iscnId`